hikari/api/special_endpoints.py

```diff
@@ -23,7 +23,13 @@
 """Special additional endpoints used by the REST API."""
 from __future__ import annotations
 
-__all__: typing.List[str] = ["TypingIndicator", "GuildBuilder"]
+__all__: typing.List[str] = [
+    "CommandBuilder",
+    "TypingIndicator",
+    "GuildBuilder",
+    "CommandResponseBuilder",
+    "InteractionResponseBuilder",
```

Comment:
Is special endpoints the right place for these response builders or might it start to get messy in the future?